recipe-server/client/control_new/tests/components/recipes/test_SwitchBox.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import SwitchBox from 'control_new/components/forms/SwitchBox';
+
+describe('<SwitchBox>', () => {
+  const props = {
+    children: null,
+    onChange: () => {},
+    value: null,
+  };
+
+  it('should work', () => {
+    const wrapper = () => shallow(<SwitchBox {...props} />);
+
+    expect(wrapper).not.toThrow();
```

Comment:
We could probably write a more in depth version of this test. Maybe checking that clicking acts as expected, and that the children get rendered?